generator/test/test_util.dart

```diff
@@ -0,0 +1,64 @@
+import 'dart:io';
+import 'dart:async';
+import 'dart:convert';
+import 'package:path/path.dart' as _path;
+import 'package:dart_style/dart_style.dart';
+import 'package:build/build.dart';
+import 'package:build_test/build_test.dart';
+import 'package:realm_generator/realm_generator.dart';
+
+String _stringReplacements(String content) {
+  final formatter = DartFormatter();
+  var lines = LineSplitter.split(content);
+  String formattedContent = lines.where((element) => !element.startsWith("part of")).join('\n');
```

Comment:
@nielsenko  do you it would be correct to generate also `part of ` ? Currently I remove programmatically this line from expected file output in order to match with the actual result.

Reply:
As you may have noticed the `part of` is declared when you run the generator. It is just done by the `source_gen|combining_builder` that is setup in `build.yaml`. This allows us to co-exists with other builders that output to the same file.

Reply:
OK